share/man/man1/brew.1

```diff
@@ -554,10 +554,6 @@ HOMEBREW_BROWSER
 If set, uses this setting as the browser when opening project homepages, instead of the OS default browser\.
 .
 .TP
-HOMEBREW_BUILD_FROM_SOURCE
```

Comment:
> Why does it need to be documented in the manpage?

Make it friendly to people who need such custom to find it. It's like that there is a custom option which will have lower support than default one. But if you need it, here it is.  :wink: